src/lib/style/Portal.scss

```diff
@@ -581,4 +582,142 @@ li.SRC-table-anchor-chosen, li.SRC-table-anchor-chosen > a {
 
 .SRC-light-background {
   background: #F5F5F5;
-}
\ No newline at end of file
+}
```

Comment:
it was originally a copy of Portal.scss, but has now turned into all scss necessary for the components, it will be good to break this down in the near future to smaller files that are grouped by relevance..